src/main/scala/it/pagopa/interop/agreementmanagement/server/impl/Dependencies.scala

```diff
@@ -116,7 +118,7 @@ trait Dependencies {
   ): AgreementApi = new AgreementApi(
     AgreementApiServiceImpl(actorSystem, sharding, agreementPersistenceEntity, uuidSupplier, dateTimeSupplier),
     AgreementApiMarshallerImpl,
-    jwtReader.OAuth2JWTValidatorAsContexts
+    jwtReader.OAuth2JWTValidatorAsContexts(Logger.takingImplicit[ContextFieldsToLog]("OAuth2JWTValidatorAsContexts"))
```

Comment:
could it be worth to define an implicit val at the beginning?
I'm thinking about services that have multiple APIs classes that would probably use that approach.
This way we could have an homogeneous solution

Reply:
Yep, it might be a good fit. All the next PRs will have this defined as an implicit val